tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -94,6 +95,31 @@ void GenerateIntegrationTestsWindowsMatrix(TargetFramework[] targetFrameworks)
                 AzurePipelines.Instance.SetVariable("integration_tests_windows_matrix", JsonConvert.SerializeObject(matrix, Formatting.None));
             }
 
+            void GenerateIntegrationTestsWindowsAzureFunctionsMatrix()
+            {
+                // TODO: test on both x86 and x64?
+                const string v3Install = @"choco install azure-functions-core-tools-3 --params ""'/x64'""";
+                const string v3Uninstall = @"choco uninstall azure-functions-core-tools-3";
+                // const string v4Install = @"choco install azure-functions-core-tools --params ""'/x64'""";
```

Comment:
> does using azure-functions-core-tools with no suffix mean that we will automatically upgrade to v5 when it's released

Maybe 🤷‍♂️ Possibly? There's no talk of V5 at this point, and there's no `-4` suffix tools we could use, so I think we just cross that when we come to it!

> do the install log the version we use in the test in case it starts failing after an update?

Yup, that's what the call to `func` does [in Zach's comment](https://github.com/DataDog/dd-trace-dotnet/pull/3422#discussion_r1016000064) 🙂